src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -228,7 +231,8 @@ function ReportPreview({
        In this case, we don't want to show the merchant because it says: "Pending route...", which is already displayed in the amount field.
      */
     const shouldShowSingleRequestMerchant = numberOfRequests === 1 && !!formattedMerchant && !(hasOnlyTransactionsWithPendingRoutes && !totalDisplaySpend);
```

Comment:
Can we just change this line?
```suggestion
    const shouldShowSingleRequestMerchantOrDescription = numberOfRequests === 1 && (!!formattedMerchant || !!formattedDescription) && !(hasOnlyTransactionsWithPendingRoutes && !totalDisplaySpend);
```